### ISSUE TITLE
[BUG] v3.x Saving User in Manager doesn't fire the same event as v1.4

### ISSUE THREAD

**Issue by BBloke** [evocms-community/evolution#47]
When saving a user in the manager.  The event fired doesn't match v1.4.

This prevents plugins which worked in v1.4 from continuing to work in v3.x

**Comment by Pathologic** [comment]
What event do you mean?

**Comment by BBloke** [comment]
When saving a user in the manager. The follow events are fired in the different versions of the program.

v1.4 it is OnWebSaveUser
v3.1 it is OnUserFormSave

**Comment by Pathologic** [comment]
Events in 3.x are: OnUserSave and OnBeforeUserSave. It's fixed in 3.1.13:
https://github.com/evocms-community/UserManager/commit/181cbbef5a0a32ce19643b6fe2c55f9016fe3e7b
https://github.com/evocms-community/UserManager/commit/65d7fb24af51546ec15d415c7e02c3368c024d26

If you want to create a plugin to be working both in 3.x and 1.4.x, then you should check for all events:
```
if ($modx->event->name == 'OnWebSaveUser' or $modx->event->name == 'OnSaveUser') {
...
}
```
